app/views/recipes/show.html.haml

```diff
@@ -4,12 +4,14 @@
 %p Style: #{@recipe.style}
 %p Batch Size: #{@recipe.batch_size} gal.
 %p Original Gravity: #{'%.3f' % @recipe.original_gravity}
+- if @recipe.yeasts.first
+  %p Final Gravity: #{'%.3f' % @recipe.final_gravity || ''}
```

Comment:
Consider adding a `format_gravity(gravity)` helper method in `RecipesHelper` to be used on line 6 and 8 in this template.